src/Components/Blazor/Server/src/Services/BlazorHostingServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net.Mime;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.StaticFiles;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class BlazorHostingServiceCollectionExtensions
+    {
+        public static IServiceCollection AddBlazorStaticFilesConfiguration(this IServiceCollection services)
```

Comment:
Should this be `AddBlazorWebAssembly`?

Reply:
This is intentionally a horrible name to ensure we fix it before preview2. As it is stated on the PR description this method will go away in favor of something like `endpoints.MapClientSideBlazor` or something like that.

Reply:
Can you file an issue to track fixing it if it's not part of this PR?

Reply:
https://github.com/dotnet/aspnetcore/issues/19060

Reply:
> to ensure we fix it before preview2

When are you expecting to make design decisions for this? If it's going to be implemented as part of preview 2, it needs to be in the next couple of days.

Reply:
With the work to support compression.